### ISSUE TITLE
Release query-adapter

### ISSUE THREAD

**Issue by igorkamyshev** [yumauri/effector-storage#17]
Hello! It is a great lib, thanks for your work. I try to use query-adapter to save my store value in query-string, but it doesn't exist in npm-version. Could you release it, please?

It is the last version of package:
<img width="476" alt="Screenshot 2021-03-31 at 11 37 13" src="https://user-images.githubusercontent.com/26767722/113091113-72a02b00-9215-11eb-9b7f-3b5fe42ee4ac.png">

And I cannot see `query` folder in directory:
<img width="1032" alt="Screenshot 2021-03-31 at 11 36 34" src="https://user-images.githubusercontent.com/26767722/113091094-661bd280-9215-11eb-8eb4-6f9924b62c6b.png">

**Comment by yumauri** [comment]
Hello!

Yeah, I didn't release version with query string adapter, because while using it locally I've encountered with not a nice behaviour:

If you persist two (for example) stores in query string, and those stores updates (almost) simultaneously — it causes two updates of query string. Thus, if you use "Back" button later — you have to click it twice to return to logically previous state, because there are two history records...
I wanted to add some sort of debouncing + batching, but didn't have time since.

You can try it though, simply by copying adapter source code, and using it with root `persist` function with option `adapter`.

Or I can release `@next` version, so you can try it :)

**Comment by igorkamyshev** [comment]
Maybe, we can add options for this adapter to use `replaceState`? It can fix this issue.

**Comment by igorkamyshev** [comment]
And at least, this situation should be noticed in documentation 🤗

**Comment by yumauri** [comment]
[There is such option](https://github.com/yumauri/effector-storage/blob/master/src/query/README.md#url-change-method) in the adapter :)
```js
import { persist, replaceState } from 'effector-storage/query'

// use `history.replaceState` to change query string
persist({ store: $id, key: 'id', method: replaceState })
```
But this is not a fix, this is just avoiding the problem (and acquiring new one — impossibility to use "Back" button).

But I agree, I will make a remark in documentation and release new version :)

Maybe later someone will help me with PR ;)

**Comment by igorkamyshev** [comment]
Wow, my fault, sorry. Yes, you're right, it isn't solution.

**Comment by yumauri** [comment]
I've published version 4.2.0, with query string adapter, you can try it :)

There is also one more new adapter — `memory`, but no documentation for it yet :) It just persist store value in Map, and can be used to _indirectly_ tie several stores in the application to each other (because stores, persisted with the same adapter, with the same `keyArea`, with the same `key` become synchronized).

Also I've decided to rename `.cjs.js` files back to `.cjs`, to fix issue #16. This shouldn't break any other bundlers, aside from FuseBox 3, but I'll test this change nonetheless later today.

**Comment by igorkamyshev** [comment]
Thanks!
